Failed to run Godwoken custom build

error: failed to run custom build command for `gw-types v0.1.0 (/home/runner/work/godwoken-tests/godwoken-tests/kicker/packages/godwoken/crates/types)`

Caused by:
  process didn't exit successfully: `/home/runner/work/godwoken-tests/godwoken-tests/kicker/packages/godwoken/target/debug/build/gw-types-9e94f242bfcad04e/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=schemas/blockchain.mol

  --- stderr
  thread 'main' panicked at 'run rustfmt', crates/types/build.rs:43:13
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
make[2]: *** No rule to make target 'copy-godwoken-binary-from-packages-to-workspace'.  Stop.
make[1]: *** [Makefile:68: install] Error 2
make: *** [Makefile:57: init] Error 2
Error: Process completed with exit code 2.

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -192,7 +192,6 @@ jobs:
           sed -i \
             -e "s|MANUAL_BUILD_GODWOKEN=false|MANUAL_BUILD_GODWOKEN=true|g" \
             -e "s|GODWOKEN_GIT_CHECKOUT=.*|GODWOKEN_GIT_CHECKOUT=${{ github.event.inputs.godwoken }}|g" \
-            -e "s|BUILD_GODWOKEN_ON_LOCAL_OVER_DOCKER=false|BUILD_GODWOKEN_ON_LOCAL_OVER_DOCKER=true|g" \
             .build.mode.env
         fi
         if [ -n "${{ github.event.inputs.godwoken-scripts }}" ]; then

